contents/swift/transition-to-swift.md

```diff
@@ -0,0 +1,309 @@
+# Swift
+
+Authors: Ch'ng Ming Shin
+
+# Overview
+
+If you are transitioning from another language to Swift, I am here to make your life easier. The common solutions to "How to learn Swift" from Quora and StackOverflow are to
+
+1. Dive into a project
+
+2. Read the official documentation
+https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/index.html
+
+
+Those are great suggestions but diving into a new project with no prior experience to Swift makes it an ardous trial of google searches of how to implement stuff. And reading documentation, while really helpful, might be overkill and boring as there might be many concepts that you already know.
+
+To ease this pain, I am providing a 3rd solution where I try to summarise the more common and important aspects of Swift that are generally different from other languages.
```

Comment:
I am not sure whether this needs to be stated, because it is kind of implied (since that's the objective of having [se-edu#learningresources](https://github.com/se-edu/learningresources)).